kolibri_instant_schools_plugin/assets/src/views/sign-up-page/index.vue

```diff
@@ -348,14 +363,19 @@
     width: ($iphone-5-width - 20)px
 
   .terms
-    background-color: $core-bg-light
-    color: $core-text-annotation
-    height: 6em
-    overflow-y: scroll
-    padding: 0.5em
-    margin-bottom: 1em
-    p
-      margin-top: 0
+    height: 80vh
+    width: 80vw
+    &-agreement
+      &-view-prompt
```

Comment:
Let's get rid of the left and right padding, and add some top & bottom margin of 16px?

Reply:
What padding? On the button? A lot of the core styles ended up wiping the button clean locally - what's it look like for you?

Reply:
![image](https://user-images.githubusercontent.com/7193975/32353170-8abaea9a-bfe1-11e7-99ab-07f358486007.png)

Reply:
Cleared all that from button! used 1rem instead of 16px for consistency with k-textbox. You feel strongly about 16?

Reply:
Nope, I'm pretty sure 1 rem is 16px here  so that works too!